src/Compilers/Core/Portable/Compilation/CompilationOptions.cs

```diff
@@ -46,11 +46,11 @@ public abstract class CompilationOptions
         // such value is currently not serializable by JSON serializer.
 
         /// <summary>
-        /// Specifies public key used to generate strong name for the compilation assembly, or empty of not specified.
+        /// Specifies public key used to generate strong name for the compilation assembly, or empty if not specified.
         /// </summary>
         /// <remarks>
         /// If specified the values of <see cref="CryptoKeyFile"/> and <see cref="CryptoKeyContainer"/> must be null.
-        /// If <see cref="DelaySign"/> is false the assembly is marked as signed but not actually signed (aka "OSS signing").
+        /// If <see cref="PublicSign"/> is true the assembly is marked as signed but not actually signed (aka "OSS signing").
```

Comment:
Perhaps we should say "not actually signed with a private key" rather than "not actually signed"?